Sources/Executable/main.swift

```diff
@@ -45,8 +45,8 @@ do {
             "Commands to help manage providers."
         ]),
     ], arguments: Array(iterator), help: [
-        "Join our Slack if you have questions, need help,",
-        "or want to contribute: http://vapor.team"
```

Comment:
vapor.team will actually start pointing to discord soon (just waiting on TTL). but the other change is good! we should probably just change that to say "team chat" really.